docs/platform-integration/communication/sms.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "SMS"
+description: "Learn how to use the .NET MAUI Sms class in Microsoft.Maui.ApplicationModel.Communication to open the default SMS application. The text message can be preloaded with a message and recipient."
+ms.date: 05/11/2022
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.ApplicationModel.Communication"]
+---
+
+# SMS
```

Comment:
@davidbritch Do you think so? I mentally fought on this because who, even in the developer audience, knows what SMS stands for? I was thinking of changing to Text messaging 😁 But I think the two most common search terms would be "SMS" and "text message(ing)"

Ideas?

Reply:
Leave as it is then. Now I've thought about it, one ever commented on it in the Xamarin.Essentials docs.